wrench/src/yaml_helper.rs

```diff
@@ -239,4 +262,11 @@ impl YamlHelper for Yaml {
             }
         }
     }
+
+    fn as_mix_blend_mode(&self) -> Option<MixBlendMode> {
+        return match self.as_str() {
```

Comment:
Huh, didn't know that Option had helper methods. That's nice!

Map wasn't working for me because string_to_mix_blend_mode returns an Option also. But it looks like and_then will do the job.